traffic_control/clients/python/trafficops/__version__.py

```diff
@@ -21,17 +21,10 @@
 client to grow in a version-controlled manner without being tied to the release cadence of Apache
 Traffic Control as a whole.
 
-Version 1.0 is supported for use with Apache Traffic Control version 3.0 (release pending at the
+Version 2.0 is supported for use with Apache Traffic Control version 4.0 (release pending at the
```

Comment:
it wont work with ATC 4.0 but 4.1 right?

Reply:
oh, right. API 2.0 isn't in ATC 4.0.